### PR TITLE
Add template for PgWeb

### PR DESCRIPTION
**Pull request by arajkumar**

### CODE REVIEW

File: openshift/pgweb-template.yaml [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+  displayName: Docker registry
+  required: true
+  name: DOCKER_REGISTRY
+  value: "docker.io"
+
+- description: Docker image to use
+  displayName: Docker image
+  required: true
+  name: DOCKER_IMAGE
+  value: "sosedoff/pgweb"
```
Comment (miteshvp) [inline openshift/pgweb-template.yaml]: It's better to rely on our own set of images. No issues at present, but we should use an image from quay.io or our own org under docker.io

File: README.md [hunk 2]
```diff
@@ -2,6 +2,8 @@
 
 This repository contains Dockerfile for PgBouncer for Fabric8-Analytics project.
 
+It also contains a openshift template for [PgWeb](https://github.com/sosedoff/pgweb), which would useful for DB visualization.
```
Comment (miteshvp) [inline README.md]: Can you also add where pgweb will come up (http://localhost:8081) or something else?

Comment (arajkumar) [reply README.md]: Thanks @miteshvp. Added more info.

File: openshift/pgweb-template.yaml (hunk 1 above)
Comment (arajkumar) [reply openshift/pgweb-template.yaml]: Since it is meant for debugging, I thought it is fine to pull it from outside. Is it fine to do it latter?

Comment (miteshvp) [reply openshift/pgweb-template.yaml]: Yes, that's absolutely fine.

**Pull request merged**